doc/tutorials/transformer/te_init.ipynb

```diff
@@ -0,0 +1,283 @@
+{
```

Comment:
<p>In another ticket, I would replace this with the python classes and also move it to the notebook connector and directly combine it with secret store.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-sandbox/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>